gulpfile.js

```diff
@@ -1290,6 +1303,10 @@ function compileJs(srcDir, srcFilename, destDir, options) {
               return enableLocalTesting(unminifiedRuntimeEsmTarget);
             } else if (destFilename === 'integration.js') {
               return enableLocalTesting(unminified3pTarget);
+            } else if (destFilename === 'amp-shadow.js') {
+              return enableLocalTesting(unminifiedShadowRuntimeTarget);
```

Comment:
could maybe just store these name pairs in a an array. 

`let enableForTesting = [{unminified: 'amp.js', minified: 'v0.js'}, ...]`;

Reply:
feel free to punt btw.

Reply:
Hah, yeah, I was going to do so! This is a quick fix to unblock @jpettitt who is trying to debug the shadow runtime.